presto-main/src/main/java/io/prestosql/sql/analyzer/Analyzer.java

```diff
@@ -84,14 +87,24 @@ public Analysis analyze(Statement statement, boolean isDescribe)
 
         // check column access permissions for each table
         analysis.getTableColumnReferences().forEach((accessControlInfo, tableColumnReferences) ->
-                tableColumnReferences.forEach((tableName, columns) ->
-                        accessControlInfo.getAccessControl().checkCanSelectFromColumns(
-                                accessControlInfo.getSecurityContext(session.getRequiredTransactionId(), session.getQueryId()),
-                                tableName,
-                                columns)));
+                tableColumnReferences.forEach((tableName, columns) -> {
+                    // TODO why getTableHandle sometimes gives empty()? (eg in TestAnalyzer.testExistingRecursiveView)
```

Comment:
I think we need to fix `TestAnalyzer` instead.

Reply:
fixing in Analyzer was the easiest what I could do (1 place); the only alternative was fixing StatementAnalyzer and Analysis (2 places)